code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1,4 +1,4 @@
-/*
+/paradise_schema.sql/*

```

Comment:
no i see what it is...i accidently pasted it in while i was doing a file search....DOH